Source/Charts/Formatters/DefaultValueFormatter.swift

```diff
@@ -94,14 +94,7 @@ open class DefaultValueFormatter: NSObject, IValueFormatter
                              dataSetIndex: Int,
                              viewPortHandler: ViewPortHandler?) -> String
     {
-        if block != nil
-        {
-            return block!(value, entry, dataSetIndex, viewPortHandler)
-        }
-        else
-        {
-            return formatter?.string(from: NSNumber(floatLiteral: value)) ?? ""
-        }
+        return block?(value, entry, dataSetIndex, viewPortHandler) ??
```

Comment:
I think `if let` would be more readable in this case.

Reply:
I flip-flopped on this one a lot. I'll change it now.